cmd/karmadactl/karmadactl.go

```diff
@@ -1,16 +1,16 @@
 package main
 
 import (
-	"os"
-
 	"k8s.io/component-base/cli"
 	_ "k8s.io/component-base/logs/json/register" // for JSON log format registration
```

Comment:
I think the comment `for JSON log format registration` not clear enough, can you give more detail info to explain why we need it?

Reply:
Yes, we don't need this in `karmadactl`, I will remove it. Thanks for reminds.

Reply:
Hi, @RainbowMango @lonelyCZ   It seems I blocked the pull request for missed the question that @RainbowMango put forward.
 Sorry for that .
`1` : Import `_ "k8s.io/component-base/logs/json/register"` can support the  output the log with `json` format  by using flags `--logging-format=json`.  Default is `text` format.

![image](https://user-images.githubusercontent.com/16109961/179392632-80021901-1b04-4c95-aff6-ab7ecd1384d3.png)

`2`:  The flags is not work now because I have made a mistake and not applied the log options in pull request [2049](https://github.com/karmada-io/karmada/pull/2049) .   
Do you think should we create a issue or pr to resolve the .

Reply:
> Do you think should we create an issue or pr to resolve it.

Yes, I think it is useful.

Reply:
[2242](https://github.com/karmada-io/karmada/issues/2242) I made an issue for tracing the `json log register`.
And this pr looks good to me.